services/src/messaging/src/main/java/org/openkilda/messaging/info/InfoMessage.java

```diff
@@ -76,9 +76,9 @@ public InfoMessage(@JsonProperty(PAYLOAD) final InfoData data,
      * @param timestamp     timestamp value
      * @param correlationId message correlation id
      */
-    public InfoMessage(@JsonProperty(PAYLOAD) final InfoData data,
-                       @JsonProperty(TIMESTAMP) final long timestamp,
-                       @JsonProperty(CORRELATION_ID) final String correlationId) {
+    public InfoMessage(final InfoData data,
```

Comment:
Why remove @JsonProperty?

Reply:
the same is here (please take a look at the comment bellow)